docs/using-cinder-csi-plugin.md

```diff
@@ -423,6 +423,15 @@ Volumes:
 
 ```
 
+Additional Supported parameters in `pod`'s volume definition:
+* `type`: The `volume type` defined in cinder.
+
+```
+volumeAttributes:
+  capacity: 1Gi # default is 1Gi
+  type: magnumtype
+```
```

Comment:
Please indent the code section, under the item `type`

Reply:
done